Linked Stacks Documentation (#823)
hashicorp/terraform-docs-common: website/docs/cloud-docs/stacks/deploy/plans.mdx

* The number of your stack’s components and deployments  
* Your deployment’s current [health](#check-deployment-health)  
* The latest configuration version
* The inputs and outputs for your stack   
* A chart listing each deployment’s recent configuration versions

> Clicking on a Stack’s **Inputs** reveals a list of the inputs this Stacks is using, along with the upstream Stacks that supply those inputs. Clicking on a Stack’s **Outputs** reveals a list of the outputs of this Stack, and any downstream Stacks that intake those outputs. To learn more about passing data from one Stack to another, refer to [Pass data from one Stack to another](/terraform/language/stacks/deploy/link-stacks).

Reviewer: the stack inputs page currently represents the inputs sourced from an upstream stack, it does not list out inputs from other sources. In the case where all the inputs are sourced from upstream stacks then the list represents all the stack's input. If some inputs are sourced from somewhere else, then the input list will be a partial list of the stack's input.

Author: Gotcha, I see now I was too vague in this description - thank you for catching that!!

 @Uk1288  Would saying something like:
```suggestion
Clicking on a Stack’s **Inputs** reveals a list of the inputs this Stack has access to from `upstream_input` blocks, along with the upstream Stacks that supply those inputs. Clicking on a Stack’s **Outputs** reveals a list of the outputs of this Stack and any downstream Stacks that intake those outputs. To learn more about passing data from one Stack to another, refer to [Pass data from one Stack to another](/terraform/language/stacks/deploy/link-stacks).
```

Make more sense to you? Or be clearer?

Reviewer: yes, exactly right!

You can see a plan’s full details by clicking the plan’s name or an abbreviated list of the plan’s changes by clicking **Quick View**. You can also click **View all plans** to display a list of all the plans for this deployment. You can filter plans by [health status](#check-deployment-health) in the list of all plans. 

> Each plan includes the Stack’s inputs and outputs, a timeline detailing when the plan started, when the plan received its configuration version, and when it was approved.  

Reviewer: A plan belongs to a single deployment within a stack, so the outputs of plan might not necessary represent the entire stack output. It represents the output of a single deployment.  Same with inputs, the inputs consumed by a deployment plan may be a subset of the entire stack input.

Author: Thank you for explaining that context, so this would be more accurate, correct? 
```suggestion
Each plan includes that plan's inputs and outputs, a timeline detailing when the plan started, when the plan received its configuration version, and when it was approved.  
```

Also I'd love to know if you feel like I'm missing anything else here in terms of conveying where else in the UI inputs and outputs show up?

Reviewer: this update looks good to me
